File: app/src/main/java/org/rm3l/ddwrt/exceptions/SpeedTestException.java
package org.rm3l.ddwrt.exceptions;

import android.support.annotation.Nullable;

/**
 * Created by rm3l on 18/01/16.
 */
public class SpeedTestException extends DDWRTCompanionException {
    public SpeedTestException() {
    }

    public SpeedTestException(@Nullable String detailMessage) {
        super(detailMessage);
    }

    public SpeedTestException(@Nullable String detailMessage, @Nullable Throwable throwable) {
        super(detailMessage, throwable);
    }

    public SpeedTestException(@Nullable Throwable throwable) {
        super(throwable);
    }
}

File: app/src/main/java/org/rm3l/router_companion/exceptions/SpeedTestException.kt
package org.rm3l.router_companion.exceptions

/**
 * Created by rm3l on 18/01/16.
 */
class SpeedTestException : DDWRTCompanionException {
    constructor()

    constructor(detailMessage: String?) : super(detailMessage)

    constructor(detailMessage: String?, throwable: Throwable?) : super(detailMessage, throwable)

    constructor(throwable: Throwable?) : super(throwable)
}
